.github/workflows/cloudflare-open-next-build.yml

```diff
@@ -0,0 +1,64 @@
+# Security Notes
+# Only selected Actions are allowed within this repository. Please refer to (https://github.com/nodejs/nodejs.org/settings/actions)
+# for the full list of available actions. If you want to add a new one, please reach out a maintainer with Admin permissions.
+# REVIEWERS, please always double-check security practices before merging a PR that contains Workflow changes!!
+# AUTHORS, please only use actions with explicit SHA references, and avoid using `@master` or `@main` references or `@version` tags.
+
+name: Cloudflare OpenNext Build
+
+on:
+  push:
+    branches:
+      - main
+  pull_request_target:
+    branches:
+      - main
+    types:
+      - labeled
+
+defaults:
+  run:
+    # This ensures that the working directory is the root of the repository
+    working-directory: ./
+
+permissions:
+  contents: read
+  actions: read
+
+env:
+  # See https://turbo.build/repo/docs/reference/command-line-reference/run#--cache-dir
+  TURBO_ARGS: --cache-dir=.turbo/cache
+  # See https://turbo.build/repo/docs/reference/command-line-reference/run#--force
+  TURBO_FORCE: true
+
+jobs:
+  build-cloudflare-worker:
+    name: Build Cloudflare Worker (on ubuntu)
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Harden Runner
+        uses: step-security/harden-runner@0634a2670c59f64b4a01f0f96f84700a4088b9f0 # v2.12.0
+        with:
+          egress-policy: audit
+
+      - name: Git Checkout
+        uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
+
+      - name: Set up pnpm
+        uses: pnpm/action-setup@a7487c7e89a18df4991f7f222e4898a00d66ddda # v4.1.0
+        with:
+          cache: true
+
+      - name: Set up Node.js
+        uses: actions/setup-node@49933ea5288caeca8642d1e84afbd3f7d6820020 # v4.4.0
+        with:
+          # We want to ensure that the Node.js version running here respects our supported versions
+          node-version-file: '.nvmrc'
+          cache: 'pnpm'
+
+      - name: Install packages
+        run: pnpm install --frozen-lockfile
```

Comment:
```suggestion
        run: pnpm install --frozen-lockfile --prod
```

Reply:
Will this work? `@opennextjs/cloudflare` is a dev dependency after all 🤔

Reply:
I tested this locally and indeed it doesn't work:
![Screenshot 2025-05-13 at 15 38 18](https://github.com/user-attachments/assets/aeeda220-49e5-4d96-829a-19837414323d)

Reply:
I suppose we can leave it as is, and cross that bridge when we get there

Reply:
is there any issue with not including `--prod`? 🤔

Reply:
I was under the impression that this _should_ build in production mode (similar to how next does)–but since we aren't relying on OpenNext for deployment, it's not really a problem.

Reply:
Yes, it does build in production mode, but it does so using some non-production dependencies, so I _feel_ like this is fine 🤔

I am guessing that this works with Next because in a sense the next package is both a dev (it builts your app) and production dependency (you can import stuff from next)

Reply:
Ah! actually! The OpenNext adapter is the same, you can import from it ([`getCloudflareContext`](https://opennext.js.org/cloudflare/bindings#how-to-access-bindings-in-your-nextjs-app)), Victor indeed did somewhat recently update the [OpenNext getting started doc](https://opennext.js.org/cloudflare/get-started#1-install-opennextjscloudflare) to instruct users to install the adapter as a proper dependency, if you prefer that I can just make the adapter a proper dependency 🙂 (and add here the `--prod` flag)

Reply:
Let’s keep as a dev dependency until it needs to be a production one, at least for now.